### PR TITLE
Tweaks on nav2 header

### PR DESCRIPTION
**Pull request by cecileboucheron**
@keybase/react-hackers

### CODE REVIEW

File: shared/chat/conversation/info-panel/index.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const listStyle = {
     : {
         backgroundColor: globalColors.white,
         borderLeft: border,
-        marginTop: -4 /* Necessary fix: adds 1px at the top so we hide the gray divider */,
+        marginTop: -5 /* Necessary fix: adds 1px at the top so we hide the gray divider */,
```
Comment (buoyad) [inline shared/chat/conversation/info-panel/index.js]: I think this style should be removed, `clickCatcherStyle` in `info-panel/container` moves the panel down and this is moving it back up again. `clickCatcherStyle` can just not move it down as much.

File: shared/chat/header.desktop.js [hunk 2]
```diff
@@ -105,16 +118,18 @@ const Header = (p: Props) => {
 }
 
 const styles = Styles.styleSheetCreate({
+  actionIcons: {
+    paddingBottom: Styles.globalMargins.xsmall - 4,
```
Comment (buoyad) [inline shared/chat/header.desktop.js]: `tiny`?

File: shared/chat/header.desktop.js [hunk 3]
```diff
@@ -105,16 +118,18 @@ const Header = (p: Props) => {
...
+  },
   container: {
     flexGrow: 1,
-    height: 40,
+    height: 40 - 1,
   },
   desc: Styles.platformStyles({isElectron: Styles.desktopStyles.windowDraggingClickable}),
-  grow: {flexGrow: 1},
+  headerTitle: {flexGrow: 1, paddingBottom: 4},
```
Comment (buoyad) [inline shared/chat/header.desktop.js]: `xtiny`

File: shared/teams/team/header/index.js [hunk 4]
```diff
@@ -109,15 +109,15 @@ const _TeamHeader = (props: Props) => (
...
+              size={16}
             />
           </Kb.Button>
         )}
         {props.canManageMembers && (
           <Kb.Button
             type="Default"
             mode="Secondary"
-            label={'Add people...'}
+            label={'Add people'}
```
Comment (buoyad) [inline shared/teams/team/header/index.js]: `{'Add people'}` -> `"Add people"`

File: shared/wallets/nav-header/index.js [hunk 5]
```diff
@@ -25,7 +25,7 @@ export const HeaderTitle = (props: HeaderTitleProps) => (
           {props.accountName}
         </Kb.Text>
       </Kb.Box2>
-      <SmallAccountID accountID={props.accountID} />
+      <SmallAccountID textType="BodyTiny" accountID={props.accountID} style={styles.accountID} />
```
Comment (buoyad) [inline shared/wallets/nav-header/index.js]: flow is complaining about `styles.accountID`

File: shared/wallets/export-secret-key/index.js [hunk 6]
```diff
@@ -33,7 +33,7 @@ export default class ExportSecretKeyPopup extends React.Component<Props> {
         {this.props.accountName ? (
           <Text type="BodySmallSemibold">{this.props.accountName}</Text>
         ) : (
-          <SmallAccountID accountID={this.props.accountID} />
+          <SmallAccountID textType="BodySmall" accountID={this.props.accountID} />
```
Comment (buoyad) [inline shared/wallets/export-secret-key/index.js]: `SmallAccountID` doesn't take a `textType` prop

**Comment by cecileboucheron** [comment]
@buoyad all updated and flow fixed ^

**Comment by buoyad** [comment]
Want to add headerExpandable to wallet headers too?

**Pull request merged**